DigitalLearningSolutions.Web/Controllers/ForgotPasswordController.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers
+{
+    using Microsoft.AspNetCore.Mvc;
+    using DigitalLearningSolutions.Web.ViewModels.ForgotPassword;
+    using DigitalLearningSolutions.Web.ViewModels.LearningSolutions;
+
+    public class ForgotPasswordController : Controller
+    {
+
+        [Route("/ForgotPassword")]
+        public IActionResult ForgotPassword()
+        {
+            var model = new ForgotPasswordViewModel();
+            return View(model);
+        }
+
+        [HttpPost]
+        public IActionResult SubmitEmail(string emailAddress)
+        {
+            //Do some validation
```

Comment:
We usually add a ticket reference and a TODO for upcoming work, as:
* There are a lot of tools that detect comments with a string "TODO" in them and summarise outstanding TODOs in the codebase.
* Adding a JIRA ticket reference helps with tracking work (e.g. whether ticket is done and someone has forgotten to remove a todo, or maybe if something has got prioritised the JIRA ticket will have more reasons on it about why it has happened)

In this case, this line could be:
// TODO: HEEDLS-354 - form submission logic